### PR TITLE
Issue 22 Tailwind and global styles

### LINKED ISSUE #22: Create global CSS styles and fonts
## Basic Information
Based on the current figma design, creating in `/client/tailwind.config.ts`(for styles) and `/client/styles.css`(for fonts)
image
## Other Information
**[Are there any additional information that we should know about?]**

### PR DESCRIPTION
**Pull request by ScottyTGB**
## Change Summary

- Roboto and Urbanist now the primary fonts for Headings and body receptively.
- Font sizes changed for every tag from h1 through to h6 and other paragraph tags.
- Color palette added as custom palette in the tailwind config and global css.

### Change Form
N/A. I believe front end stuff doesn't require tests.

## Other Information
All styles have been applied with accordance of the current style sheet provided on [figma](https://www.figma.com/design/usGd8nihwCr6YPvzWi1YyR/WAJO?node-id=470-1865).

## Side Note
package-lock.json has been slightly altered to address vulnerabilities and depreciation.

# Related issue

- Resolve #22

### CODE REVIEW

File: client/src/pages/index.tsx
Comment (ErikaKK) [inline client/src/pages/index.tsx:1]: Nice work. How about font styles for body?

File: client/tailwind.config.ts
Comment (ScottyTGB) [reply client/tailwind.config.ts:40]: I would require a second opinion on this one from the tech leads. I personally think adding identifiers in front of those names would be a bit finicky. For example writing bg-yellow-400 is a lot easier than bg-wajo-yellow-400. By doing `yellow: "#ffd700"` I've basically overridden Tailwind's default yellow of `yellow: "#facc15"`. So Tailwind's default yellow will never appear again, only ours. If I want to still use Tailwind's default yellow, only then I would distinguish them if it makes sense.

Comment (ErikaKK) [reply client/tailwind.config.ts:40]: I think you're right

Comment (yunho7687) [reply client/tailwind.config.ts:40]: > I would require a second opinion on this one from the tech leads @AarifLamat @yunho7687 @Julie-Salazar. I personally think adding identifiers in front of those names would be a bit finicky. For example writing bg-yellow-400 is a lot easier than bg-wajo-yellow-400. By doing `yellow: "#ffd700"` I've basically overridden Tailwind's default yellow of `yellow: "#facc15"`. So Tailwind's default yellow will never appear again, only ours. If I want to still use Tailwind's default yellow, only then I would distinguish them if it makes sense.

yeah, good point, overwriting it is applicable in this project

File: client/package-lock.json
Comment (yunho7687) [inline client/package-lock.json:1]: not sure why there's so many things changed here. what did you do for all these changes?  ngl, I am confused by these changes and not sure if there's any consequence once it's been changed

Comment (yunho7687) [reply client/package-lock.json:1]: @scratchclaggy yo, do you have any idea what kind of operation may lead to all these changes?

**Review by yunho7687 — CHANGES_REQUESTED** [review]
all in all, good job! will approve once figuring out the package-lock.json file

**Comment by ScottyTGB** [comment]
> all in all, good job! will approve once figuring out the package-lock.json file

I can revert the package-lock.json if needed. All I did was `npm audit fix`.

**Review by yunho7687 — APPROVED** [review]
great job! thanks mate!

**Comment by yunho7687** [comment]
It can be merged now

**Pull request merged**
